e2e/cypress/integration/links.spec.ts

```diff
@@ -0,0 +1,51 @@
+/// <reference types="cypress" />
+
+import TestContext from "../support/TestContext"
+const path = require('path')
+
+let ctx;
+let proc;
+
+let LINK = 'https://voxel51.com/'
+describe('links', () => {
+    before(async () => {
+      const FIXTURES = path.join(__dirname, '..', 'fixtures')
```

Comment:
I think one image is fine, but the repository is already quite big. Just noting that future tests should dynamically create image data, or use the zoo, etc.